rskj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -183,6 +184,13 @@ public Program(byte[] ops, ProgramInvoke programInvoke) {
         isLogEnabled = logger.isInfoEnabled();
         isGasLogEnabled =gasLogger.isInfoEnabled();
 
+        if (isLogEnabled ) {
+            gasLogger.info("WARNING! VM logging is enabled. This will the VM 200 times slower. Do not use in production.");
```

Comment:
If is warning, logger should be warning against info.
I thing if in production never should be enabled, this should be an **error** log.